peep.py

```diff
@@ -920,7 +920,11 @@ def peep_port(paths):
         if not hashes:
             print(req.req)
         else:
-            print('%s' % req.req, end='')
+            if req.original_link:
```

Comment:
@erikrose I'm sorry it took so log, but i had to skim through a lot of pip versions in order to make me more confident regarding this changes.
It looks like at some point, urls were stored in a property called `url`. However, contents of this property were inconsistent between versions of pip, that's why decided to support only versions above 6.1.0. Is that okay with you? Or do you have any suggestions about direction i should take?
Tests should pass from now on.